tests/integration/pilot/common/routing.go

```diff
@@ -452,6 +452,44 @@ spec:
 	return []TrafficTestCase{tc}
 }
 
+// useClientProtocolCases contains tests use_client_protocol from DestinationRule
+func useClientProtocolCases(apps *EchoDeployments) []TrafficTestCase {
+	var cases []TrafficTestCase
+	client := apps.PodA
+	destination := apps.PodC[0]
+	cases = append(cases,
+		TrafficTestCase{
+			name:   "use client protocol with h2",
+			skip:   apps.IsMulticluster(), // Not needed for multi cluster.
+			config: useClientProtocolDestinationRule("use-client-protocol-h1", destination.Config().Service),
```

Comment:
```suggestion
			config: useClientProtocolDestinationRule("use-client-protocol-h2", destination.Config().Service),
```
nit. Although you may want to actually use the `children` part of TrafficTestCase so there is one config applied once and then 2 calls